src/main.js

```diff
@@ -1,6 +1,7 @@
 import React from "react"
 import ReactDOM from "react-dom"
 import App from "./components/App"
+import PetTypesIndex from "./components/PetTypesIndex"
```

Comment:
Not sure why this component is being imported, when it isn't being used in anyway in this component.